src/ClientResolver.php

```diff
@@ -287,6 +287,20 @@ class ClientResolver
             'default'   => false,
             'fn'        => [__CLASS__, '_apply_suppress_php_deprecation_warning']
         ],
+        'ignore_configured_endpoint_urls' => [
+            'type'      => 'value',
+            'valid'     => ['bool'],
+            'doc'       => 'Set to true to disable configured endpoint urls.',
```

Comment:
endpoint urls that have been configured where?

Reply:
Yeah, could use some clarification here.